tests/integration/targets/zypper/tasks/zypper.yml

```
@@ -255,6 +255,91 @@
     that:
      - "stat_result.stat.exists == true"
 
+# Build and install an empty rpm with error in post script
+- name: uninstall post_error
+  zypper:
+    name: post_error
+    state: removed
+
+- name: install rpmbuild
+  zypper:
+    name: rpmbuild
+    state: present
+
+- name: clean zypper RPM cache
+  file: 
+    name: /var/cache/zypper/RPMS
+    state: absent
+
+- name: create directory
+  file:
+    path: "{{remote_tmp_dir | expanduser}}/zypper2"
```

Nit picking:
```suggestion
    path: "{{ remote_tmp_dir | expanduser }}/zypper2"
```
This would be a recommended convention to follow, but more importantly, we should try to be consisten throughout the file - see lines 282 and 287 and this one. Each line is expressing the same thing in a slightly different way.